src/anm/media/audio.js

```diff
@@ -47,7 +47,7 @@ var audioContext = getAudioContext();
  * @class anm.Audio
  */
 function Audio(url) {
-    this.url = url + audioExt;
+    this.url = /\.\w+$/i.test(url) ? url : url + audioExt;
```

Comment:
got it, thanks. then indeed, _only in this case_ (only when extension is already present), we should load audio type from media or else it will be `audio/ogg` for `.mp3` and s.o.

> we will soon start exporting audio the same way (not really soon, but soon)

good!